environments/perftest.yml

```diff
@@ -32,6 +32,12 @@ cname:
   - name: "asverify.bulkscan"
     ttl: 300
     record: "asverify.bulkscanperftest.blob.core.windows.net"
+  - name: "reformscan"
```

Comment:
Could you please add these to the end rather than in the middle of the list? 
Due to the current code structure, TF treats this as a list with index 0,1,2 and will have issues.  
TF tries to align all the indexes of the records after these and in doing so deletes and recreates the records. This might have a potential impact.

Reply:
We are basically in progress in implementing a fix soon to avoid these issues. But till then we need to be careful

Reply:
one way to check what your terraform code does is to see the plan from the ADO pipeline and currently, this PR, if merged to master, will destroy few records and recreate it again
```
7 to add, 0 to change, 5 to destroy
```

Reply:
Done - moved the new entries to the bottom